providers/os/resources/os.lr

```diff
@@ -792,6 +794,12 @@ docker.file.stage @defaults("from.name") {
   expose []docker.file.expose
 }
 
+// Dockerfile USER instructions
+docker.file.user @defaults("user") {
+  user string
```

Comment:
We will want comments on each of these since those generate the docs page for the resource